src/app/utils.ts

```diff
@@ -23,13 +25,17 @@ export const actionHook = <C extends AnyActionCreator>(actionCreator: C, body: A
       const result = next(action);
 
       if (matches(action)) {
-        const promise = boundHook(action);
-
-        if (promise) {
-          services.promiseCollector.add(promise);
+        try {
+          const promise = boundHook(action);
+          if (promise) {
+            services.promiseCollector.add(promise);
+          }
+        } catch (e) {
+          // how to get store ?
+
+          Sentry.captureException(e);
         }
```

Comment:
@TomWoodward this is where I'm stuck. The store isn't available anywhere that I can see. Where can I access it from, or should I change the arguments to add a "store" and then change all the spots where this is called from provide that?

Reply:
`stateHelpers` (defined a few lines up) has `getState` and `dispatch`